src/controllers/expenses.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+
+const expanseServise = require('../services/expenses');
+const userServise = require('../services/users');
+
+const add = (req, res) => {
+  const { userId, spentAt, title, amount, category, note } = req.body;
+
+  const foundUser = userServise.getById(userId);
+
+  const isValidData
+    = foundUser
```

Comment:
`=` shouldn't start from the new line
```suggestion
  const isValidData = foundUser
```